app/views/layouts/_footer.html.erb

```diff
@@ -1,38 +1,29 @@
-<footer class="govuk-footer" role="contentinfo">
+<footer class="govuk-footer govuk-!-display-none-print" role="contentinfo">
   <div class="govuk-width-container">
-    <% if current_user.present? && current_user["admin"] && current_user["accepted_terms?"] %>
-      <div class="govuk-footer__navigation" id="admin">
-        <div class="govuk-footer__section">
-          <h2 class="govuk-footer__heading govuk-heading-m">Support tools</h2>
-          <ul class="govuk-footer__list">
-            <li class="govuk-footer__list-item">
+    <div class="govuk-footer__meta">
+      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+        <% if current_user.present? && current_user["admin"] && current_user["accepted_terms?"] %>
+          <h2 class="govuk-heading-m" id="admin">Support tools</h2>
+          <ul class="govuk-footer__inline-list govuk-!-margin-bottom-6">
+            <li class="govuk-footer__inline-list-item">
               <%= link_to "Access Requests (#{AccessRequest.return_count})", access_requests_path, class: "govuk-footer__link", data: { qa: "access_requests_link" } %>
             </li>
-            <li class="govuk-footer__list-item">
+            <li class="govuk-footer__inline-list-item">
               <%= link_to "Active users by organisation", organisations_support_page_path, class: "govuk-footer__link", data: { qa: "organisations_link" } %>
             </li>
           </ul>
-        </div>
-      </div>
-      <hr class="govuk-footer__section-break">
-    <% end %>
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds govuk-footer__content-container">
-        <p class="govuk-footer__content">
-          Contact the Becoming a Teacher team for support:
-          <%= bat_contact_mail_to bat_contact_email_address, subject: "Publish teacher training courses support", link_class: "govuk-footer__link" %>
-        </p>
-      </div>
-    </div>
-    <div class="govuk-footer__meta">
-      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+        <% end %>
+        <h2 class="govuk-heading-m">Get support</h2>
+        <ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16 govuk-!-margin-bottom-8">
+          <li>Email: <%= bat_contact_mail_to bat_contact_email_address, subject: "Publish teacher training courses support", link_class: "govuk-footer__link" %></li>
+          <li class="govuk-!-margin-bottom-4">We respond within 5 working days, or one working day for more urgent queries</li>
+          <li><%= link_to "How to publish teacher training courses", guidance_path, class: "govuk-footer__link" %></li>
+          <li><%= link_to "Teacher Training Courses API documentation", "https://api.publish-teacher-training-courses.service.gov.uk", class: "govuk-footer__link" %></li>
```

Comment:
Should we just say "Teacher Training Courses API" here or does that break the general content guidance rules?

Reply:
Think it’s best to be clear precisely what this is. Going to merge this change now, can always revise later.